src/controllers/routers/beneficiaries.js

```diff
@@ -1,24 +1,166 @@
-/*!
- * OneSG API Server by TL Bootcamp#6 OneSG Team
- * Copyright(c) 2020 TechLadies
- * MIT Licensed
- */
-
+const {
+  ValidationError,
+  UniqueViolationError,
+  NotNullViolationError,
+} = require('objection');
+const {
+  errors: { BadRequest, InvalidInput, ResourceNotFound },
+} = require('../../utils');
+const { Beneficiary } = require('../../models');
 /**
- * Module dependencies.
+ * Sanitize data from client. Call before an insert or an update.
  */
-const { Beneficiary } = require('../../models');
+function sanitize(json) {
+  const beneficiary = json;
+  if (json.name) {
+    beneficiary.name = json.name.trim();
+  }
+  if (json.phone && !Number.isInteger(json.phone)) {
+    beneficiary.phone = json.phone.trim();
+  }
+  if (json.email) {
+    beneficiary.email = json.email.toLowerCase().trim();
+  }
+  if (json.occupation) {
+    beneficiary.occupation = json.occupation.trim();
+  }
+  if (json.householdIncome) {
+    beneficiary.householdIncome = parseFloat(json.householdIncome, 10);
+  }
+  if (json.householdSize) {
+    beneficiary.householdSize = parseInt(json.householdSize, 2);
+  }
+  if (json.notes) {
+    beneficiary.notes = json.notes.trim();
+  }
+  if (json.createdBy) {
+    beneficiary.createdBy = parseInt(json.createdBy, 10);
+  }
+  if (json.updatedBy) {
+    beneficiary.updatedBy = parseInt(json.updatedBy, 10);
+  }
```

Comment:
The `createdBy` and `updateBy` shouldn't be part of the `req.body` but interpreted from the user authentication token.